docs/standard/components.md

```diff
@@ -1,80 +1,82 @@
+The .NET Standard is a set of APIs that are implemented by the Base Class Library of a .NET implementation. More formally, it's a specification of .NET APIs that make up a uniform set of contracts that you compile your code against. These contracts are implemented in each .NET implementation. This enables portability across different .NET implementations, making it so that your code can effectively run everywhere.
 
-More formally, it is a specification of .NET APIs which make up a uniform set of contracts that you compile your code against.  These contracts have underlying implementations for each .NET runtime.  This enables portability across different .NET runtimes, making it so that your code can effectively "run everywhere".
-
-The .NET Standard is also a build target. You can currently target .NET Standard 1.0-2.0. If your code targets a version of the .NET Standard, it is guaranteed to run on any .NET runtime which implements that version.
+The .NET Standard is also a target framework. If your code targets a version of the .NET Standard, it can run on any .NET implementation which supports that version of .NET Standard.
 
 To learn more about the .NET Standard and how to target it, see the [.NET Standard](net-standard.md) topic.
 
-## .NET runtimes
+## .NET implementations
+
+Each implementation of .NET includes the following components:
 
-There are 3 primary .NET runtimes which Microsoft actively develops and maintains: .NET Core, .NET Framework, and Mono for Xamarin.
+- One or more runtimes. Examples: CLR for .NET Framework, CoreCLR and CoreRT for .NET Core.
+- A class library that implements the .NET Standard and may implement additional APIs. Examples: .NET Framework Base Class Library, .NET Core Base Class Library.
+- Optionally, one or more application frameworks. Examples: [ASP.NET](https://www.asp.net/), [Windows Forms](../framework/winforms/windows-forms-overview.md), and [Windows Presentation Foundation (WPF)](../framework/wpf/index.md) are included in the .NET Framework.
+- Optionally, development tools. Some development tools are shared among multiple implementations.
+
+There are three primary .NET implementations that Microsoft actively develops and maintains: .NET Core, .NET Framework, and Mono for Xamarin.
 
 ### .NET Core
 
-.NET Core is a cross-platform runtime optimized for server workloads.  It implements the .NET Standard, which means that any code that targets the .NET Standard can run on .NET Core.  It is the runtime used by ASP.NET Core and the Universal Windows Platform (UWP).  It is modern, efficient, and designed to handle server and cloud workloads at scale.
+.NET Core is a cross-platform implementation of .NET optimized for server workloads. It's modern, efficient, and designed to handle server and cloud workloads at scale. It implements the .NET Standard, so code that targets the .NET Standard can run on .NET Core. ASP.NET Core and the Universal Windows Platform (UWP) run on .NET Core.
```

Comment:
We need to separate .NET Core from UWP. The code is shared but there also quite different. I'd make .NET for UWP anther top level implementation and a peer to .NET Core.